ci/build_and_test.bat

```diff
@@ -14,6 +14,11 @@ if "%CFG_RELEASE_CHANNEL%"=="nightly" (
 cargo test || exit /b 1
 
 :: Build and test other crates
-cd config_proc_macro || exit /b 1
-cargo build --locked || exit /b 1
-cargo test || exit /b 1
+set crates=config_proc_macro check_diff
+(for %%crate in (%crates%)) do
+    cd %%crate || exit /b 1
+    cargo build --locked || exit /b 1
+    cargo test || exit /b 1
+    :: change back to root
+    cd .. || exit /b 1
+done
```

Comment:
CI is failing. I think it might honestly just be simpler in this case to write:

```bat
:: Build and test config_proc_macro
cd config_proc_macro || exit /b 1
cargo build --locked || exit /b 1
cargo test || exit /b 1

:: Build and test check_diff
cd ..
cd check_diff || exit /b 1
cargo build --locked || exit /b 1
cargo test || exit /b 1
```